src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2688,10 +3175,15 @@ subroutine CNBackgroundLitterfall (num_soilp, filter_soilp, &
                endif
             end if
 
+            if (use_matrixcn) then   
+               if(frootn(p) .ne. 0)then
+                  matrix_nphtransfer(p,ifroot_to_iout_phn) = frootn_to_litter(p) / frootn(p)
+               end if
+            end if !use_matrixcn
          end if
 
       end do
-
+!matrix for leafn_to_retransn will be added in allocation subroutine
```

Comment:
Does this comment mean there is something missing that needs to be added in? Or is it just saying that leafn_to_retransn matrix settings is done in the allocation subroutine? If so the allocation should only allocate and set to nan, like other arrays. If LitterFall needs to happen for leafn_to_retransn it should be here.

Reply:
Matrix for leafn_to_restransn were all calculated here. So I suppose the comment here should be "matrix for retransn_to_leafn will be added in allocation subroutine", which original code also allocate retransn in allocation subroutine. I fixed the comments.